src/frontend/data/stores/useThumbnail/index.tsx

```diff
@@ -21,14 +24,20 @@ export const [useThumbnail] = create<State>((set, get) => {
   }
 
   return {
+    addMultipleResources: (thumbnailsToAdd: Thumbnail[]) =>
```

Comment:
I think we'd rather not expose action performers on the state?

We could keep the reducers to perform the changes on the state (having one generic reducer for any kind of resource) and just expose a dispatch method.

I'd even propose getting rid of the action creators themselves to minimize boilerplate.

WDYT?